appium-uiautomator2-server/build.gradle

```diff
@@ -37,3 +37,17 @@ dependencies {
     androidTestCompile 'com.android.support.test.espresso:espresso-core:2.0'
 
 }
+
+task installAUT(type: Exec) {
```

Comment:
after adding `dependsOn: 'installDebug'`, doing `gradle installDebug` is installing only the server apk but not the app under test.